blob/src/main/java/io/crate/plugin/BlobPlugin.java

```diff
@@ -65,6 +67,10 @@ public String description() {
 
     @Override
     public Collection<Module> createGuiceModules() {
+        if (settings.getAsBoolean(Node.NODE_DATA_SETTING.getKey(), false)) {
```

Comment:
why not use:
```
if (Node.NODE_DATA_SETTING.get(settings)) {
    return Collections.emptyList();
}```
Which will give the default value (true) if it's not explicitily set.